src/components/molecules/AdminSpaceCard/AdminSpaceCard.scss

```diff
@@ -11,65 +16,110 @@ $admin-space-card-logo-size: 80px;
   flex-direction: column;
   border: none;
   border-radius: $border-radius--xl;
-  padding: font-size--root(1);
   gap: font-size--root(0.5);
-
-  .AdminShowcaseSubTitle {
-    font-weight: $font-weight--700;
-    font-size: font-size--parent(1.6);
-    color: $content--over;
-  }
+  max-height: 450px;
```

Comment:
any relation to `$admin-space-card-bg-height`? maybe 4:3 ratio? `3 * $admin-space-card-bg-height`?

Reply:
done 0ab68ac